ozaria/engine/cinematic/dialogsystem/DialogSystem.js

```diff
@@ -104,18 +104,30 @@ class SpeechBubble {
     this.id = `speech-${_id++}`
     const parser = new DOMParser()
     const html = parser.parseFromString(htmlString, 'text/html')
+
     const textDiv = html.body.firstChild
-    textDiv.style.display = 'inline-block'
-    textDiv.style.position = 'absolute'
-    textDiv.style.maxWidth = SPEECH_BUBBLE_MAX_WIDTH
-    textDiv.style.opacity = 0
-    textDiv.id = this.id
-    textDiv.className = `cinematic-speech-bubble-${side}`
+    const speechBubbleDiv = document.createElement('div')
+
+    speechBubbleDiv.style.display = 'inline-block'
+    speechBubbleDiv.style.position = 'absolute'
+    speechBubbleDiv.style.maxWidth = SPEECH_BUBBLE_MAX_WIDTH
+    speechBubbleDiv.id = this.id
+    speechBubbleDiv.className = `cinematic-speech-bubble-${side}`
+    speechBubbleDiv.style.opacity = 0
+
+    speechBubbleDiv.appendChild(textDiv)
+
+    const clickToContinue = document.createElement('div')
+    clickToContinue.className = `cinematic-speech-bubble-click-continue`
+    // TODO punted i18n for Click to Continue.
```

Comment:
Curious as to why?  Why it problematic to access the translations?

Reply:
If I manually add the translation as a property on the html it may get picked up correctly.
I punted as there are many other rougher edges that I'm running behind on.